controller/user.js

```diff
@@ -200,10 +210,40 @@ const update = async (req, res, next) => {
   }
 };
 
+const updateAvatar = async (req, res, next) => {
+  const { path: filePath } = req.file;
+  const { _id: id, email } = req.user;
+
+  try {
+    const newFileName = `${email}_avatar.jpg`;
```

Comment:
rozwaz czy nie uzyc randomowego id raczej email.

Reply:
Okej, tak zrobię, jak teraz sobie myślę to ma to więcej sensu